src/highlight/__tests__/HighlightCanvas-test.tsx

```diff
@@ -39,7 +39,6 @@ describe('HighlightCanvas', () => {
     describe('getContext()', () => {
         test.each`
             canvasRef                      | expectedResult
-            ${null}                        | ${null}
```

Comment:
Why are we removing this test case?

Reply:
Oh whoops! That change isn't supposed to be there; I'll fix it!